frontend/src/app/main/ui/workspace/comments.cljs

```diff
@@ -119,16 +115,7 @@
         (mf/use-fn
          (mf/deps page-id)
          (fn [thread]
-           (when (not= page-id (:page-id thread))
-             (st/emit! (dcm/go-to-workspace :page-id (:page-id thread)
-                                            ::rt/new-window true)))
-           (tm/schedule
-            (fn []
-              (st/emit! (when (not= page-id (:page-id thread))
-                          (dw/select-for-drawing :comments))
-                        (dwcm/center-to-comment-thread thread)
-                        (-> (dcmt/open-thread thread)
-                            (with-meta {::ev/origin "workspace"})))))))]
+           (st/emit! (dwcm/navigate-to-comment (:id thread)))))]
```

Comment:
I think, with that change you makes a RPC request on each click on the thread, independently if it is loaded or not.

Reply:
You're right, I'll fix it.